staff/victor-abadia/cv/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+</head>
+
+<body>
+    <header>
+        <h1>Eddard Stark</h1>
+    </header>
+    <br>
+    <nav>
+        <a href="#personal"><h4>Personal Data</h4></a>
+        <a href="#education"><h4>Education</h4></a>
+        <a href="#employment"><h4>Employment</h4></a>
+    </nav>
+    <br>
+    <section>
+        <section id=“personal”>
+            <h3>Personal Data</h3>
+            <p>c/Castle street, 1</p>
+            <p>Winterfell - 07070</p>
+        </section>
+        <section id=“education”>
+            <h3>Education</h3>
+            <p>2007 - Fight with sword and shield</p>
+            <p>2012 - Fight with long sword</p>
+        </section>
+        <section id=“employment”>
+            <h3>Employment</h3>
+            <table>
+                <tr>
+                    <th>Company 1</th>
+                    <td>Winterfell Army</td>
+                    <th>Title</th>
+                    <td>Soldier</td>
+                    <th>Dates</th>
+                    <td>2009-2011</td>
+                </tr>
+                <tr>
+                    <th>Company 2</th>
+                    <td>Robert's Rebellion</td>
+                    <th>Title</th>
+                    <td>Elite Swordman</td>
+                    <th>Dates</th>
+                    <td>2014-2016</td>
+                </tr>
+                <tr>
+                    <th>Company 3</th>
+                    <td>The Throne</td>
+                    <th>Title</th>
+                    <td>Hand of the King</td>
+                    <th>Dates</th>
+                    <td>2016-2019</td>
+                </tr>
+            </table>
+        </section>
+    </section>
+    <br>
+    <footer>
+        <h3>Contact</h3>
+        <p>e-mail: Eddard.Stark@7kingdoms.com</p>
```

Comment:
If you want, you can use: <a><a href="mailto:rick@pawn.com">Email</a></a> to open directly the mail client